src/hyp_map.rs

```diff
@@ -132,13 +133,16 @@ const UMODE_VA_END: u64 = UMODE_VA_START + UMODE_VA_SIZE;
 // The addresses between `UMODE_MAPPINGS_START` and `UMODE_MAPPINGS_START` + `UMODE_MAPPINGS_SIZE`
 // is an area of the private page table where the hypervisor can map pages shared from guest
 // VMs. The area is divided in slots, of equal size `UMODE_MAPPING_SLOT_SIZE`.
+// Must be multiple of 4k.
 const UMODE_MAPPING_SLOT_SIZE: u64 = 4 * 1024 * 1024;
+const_assert!(PageSize::Size4k.is_aligned(UMODE_MAPPING_SLOT_SIZE));
 
-// Start of the private U-mode mappings area.
+// Start of the private U-mode mappings area.  Must be 4k-aligned.
 const UMODE_MAPPINGS_START: u64 = UMODE_VA_END + 4 * 1024 * 1024;
```

Comment:
pre-existing, but since all these constants are technically U-mode virtual addresses, should `UMODE_VA_*` maybe be renamed to `UMODE_BINARY_*`?

Reply:
Yes, I had that patch somewhere in my tree. It is on my list to change.

Reply:
Done. It was a change I had in the upcoming (next after next) signing PR.